File: DMOJ/CCC/FergusonballRatings.cpp
// CCC '22 J2 - Fergusonball Ratings

#include <bits/stdc++.h>
using namespace std;

int n, star_players = 0;
int main() {
    cin >> n;
    for (int i = n; i--;) {
        int points, fouls, rating;
        cin >> points >> fouls;

        rating = 5 * points - 3 * fouls;
        star_players += rating > 40;
    }

    cout << star_players << (star_players == n ? "+" : "") << "\n";
}

File: DMOJ/CCC/FergusonballRatings.py
# CCC '22 J2 - Fergusonball Ratings

n = int(input())
star_players = 0
for i in range(n):
    points = int(input())
    fouls = int(input())
    rating = 5 * points - 3 * fouls

    star_players += rating > 40

if star_players == n:
    print(f"{star_players}+")
else:
    print(star_players)
